src/components/ArticleCard/index.js

```diff
@@ -75,6 +78,37 @@ class ArticleCard extends React.Component {
     );
   };
 
+  articleRating = () => {
+    const { article } = this.props;
+    const { likes, dislikes } = article.reactions;
+    return (
+      <div className="article-rating meta valign-wrapper">
+        {
+          article.avg_rating.avg_rating > 0 && (
+            <>
```

Comment:
I like it that you are using React fragments, the syntax sugar it offers is great but for readability I prefer we use the `<Fragment>` tag

Reply:
Cool. I will implement that.

Reply:
Done